packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -242,11 +241,14 @@ place .graphql files in a directory at ${schemaDirPath}`);
   fs.writeFileSync(parametersFilePath, jsonString, 'utf8');
 }
 
-function getModelConfig(project) {
-  if (project && project.config && project.config.Model && project.config.Model.BillingMode) {
-    return {
-      BillingMode: project.config.Model.BillingMode,
```

Comment:
Why were we just passing BillingMode previously and what would this chnage enable us to do?

Reply:
This was actually dead code as there has never been a value passed to the config at the path model.BillingMode. I believe this was meant to be deleted but was missed in a previous PR since this variable is actually exposed via parameters.json and not via the transform.conf.json file.